Makefile

```diff
@@ -159,6 +159,8 @@ PULL_NUMBER := $(shell echo $$CLONEREFS_OPTIONS | jq '.refs[0].pulls[0].number')
 
 # -- Variables for acceptance tests
 TEST_ACCEPTANCE_START_SBO ?= local
+TEST_ACCEPTANCE_OUTPUT_DIR ?= $(OUTPUT_DIR)/acceptance-tests
+TEST_ACCEPTANCE_ARTIFACTS ?= /tmp/artifacts
```

Comment:
Is global `/tmp` a good idea? Why not under `OUTPUT_DIR`?

Reply:
`/tmp/artifacts` is the target where the `test-acceptance-artifacts` are copied/stored. It is used by the OpenShift CI jobs to collect and archive the artifacts from.

But you are right, it might be better to default it to the output dir and set it to the `/tmp/artifacts` for the CI job via parameter.

Reply:
If this is the current configured location, please go ahead.